internal/configuration/env_config.go

```diff
@@ -211,6 +212,16 @@ func (c *EnvConfig) Verify() util.MultiError {
 	return errors
 }
 
+// GetDryRun returns if dry run is enabled or not
+func (c *EnvConfig) GetDryRun() bool {
+	callPtr, _, _, _ := runtime.Caller(0)
+	value := c.getConfigValueFromEnv(util.NameOfFunction(callPtr))
+	if strings.ToLower(value) == "true" {
```

Comment:
this can be written/simplified as `return strings.ToLower(value) == "true" `

Reply:
makes sense, fixed